mlrun/datastore/sources.py

```diff
@@ -350,12 +350,55 @@ def __init__(
         super().__init__(name, attributes=attrs, **kwargs)
 
 
+class KafkaSource(OnlineSource):
+    """
+       Sets kafka source for the flow
+       :parameter brokers: list of broker IP addresses
+       :parameter topics: list of topic names on which to listen.
+       :parameter group: consumer group. Default "serving"
+       :parameter initial_offset: from where to consume the stream. Default earliest
+       :parameter partitions: Optional, A list of partitions numbers for which the function receives events.
+       :parameter sasl_user: Optional, user name to use for sasl authentications
+       :parameter sasl_pass: Optional, password to use for sasl authentications
+    """
+
+    kind = "kafka"
+
+    def __init__(
+        self,
+        brokers="localhost:9092",
+        topics="topic",
+        group="serving",
```

Comment:
Are you sure we want these defaults ? 
seems like a left over from testing or something

Reply:
same as we have in stream 
https://github.com/mlrun/mlrun/blob/1311e51857e0d029bf0c030c06e9b67fec7d771d/mlrun/datastore/sources.py#L337
it must have some defaults